x/did/client/cli/tx.go

```diff
@@ -8,12 +8,40 @@ import (
 
 	"github.com/cheqd/cheqd-node/x/did/types"
 	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/spf13/cobra"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
 )
 
+const (
+	FlagVersionID = "version-id"
+)
+
+type DIDDocument struct {
+	Context              []string             `json:"context"`
+	ID                   string               `json:"id"`
+	Controller           []string             `json:"controller,omitempty"`
+	VerificationMethod   []VerificationMethod `json:"verificationMethod,omitempty"`
+	Authentication       []string             `json:"authentication,omitempty"`
+	AssertionMethod      []string             `json:"assertionMethod,omitempty"`
+	CapabilityInvocation []string             `json:"capabilityInvocation,omitempty"`
+	CapabilityDelegation []string             `json:"capabilityDelegation,omitempty"`
+	KeyAgreement         []string             `json:"keyAgreement,omitempty"`
+	Service              []Service            `json:"service,omitempty"`
+	AlsoKnownAs          []string             `json:"alsoKnownAs,omitempty"`
+}
+
+type VerificationMethod map[string]any
```

Comment:
What structure is expected for the verification method section in the payload? Existing examples use the property name `verificationMethodType` which is not the same as DID Core spec.

Also, is the outer property called `didDoc` necessary?